### PR TITLE
Add json struct for better struct separation in rs_ingester

### PR DESCRIPTION
**Pull request by rizenfrmtheashes**
I needed to add a struct to scoop_protocol so that I wouldn't have it sitting in multiple repos as individual structs.

### CODE REVIEW

File: scoop_protocol/scoop_protocol.go [hunk 1]
```diff
@@ -43,6 +43,11 @@ type LoadCheckResponse struct {
 	ManifestURL string
 }
 
+//Scoop healthcheck for ingester healthcheck json unmarshal
```
Comment (aasted) [inline scoop_protocol/scoop_protocol.go]: We don't generally annotate for usage, as usage will probably change faster than Scoop protocol.

File: scoop_protocol/scoop_protocol.go [hunk 2]
```diff
@@ -43,6 +43,11 @@ type LoadCheckResponse struct {
...
+type ConnError struct {
```
Comment (aasted) [inline scoop_protocol/scoop_protocol.go]: Hmm, ConnError doesn't tell me very much about this. I think this is ScoopHealthCheck

File: scoop_protocol/scoop_protocol.go [hunk 3]
```diff
@@ -43,6 +43,11 @@ type LoadCheckResponse struct {
...
+	RedshiftDBConnError, IngesterDBConnError *string
```
Comment (aasted) [inline scoop_protocol/scoop_protocol.go]: In the rest of this file, we put each field on its own line. I would suggest doing the same here.

**Comment by rizenfrmtheashes** [comment]
Understood.  Let me go fiddle with some naming in rs_ingester then because I fear I will have some naming collision issues if I do change it to a name you suggested.

**Comment by rizenfrmtheashes** [comment]
Specified changes have been done.  Merge the Pull request?

**Pull request merged**